### ISSUE TITLE
Author Metadata: {Zheng Yuan}

### ISSUE THREAD

**Issue by zhengyuan-nlp** [acl-org/acl-anthology#3416]
### Author Pages

https://aclanthology.org/people/z/zheng-yuan/

### Type of Author Metadata Correction

- [X] The author page wrongly conflates different people with the same name.
- [ ] This author has multiple pages with different spellings of their name.
- [ ] This author has permanently changed their name.

### Supporting Information

The papers listed under the author page include works by multiple individuals sharing the same name.
My Google Scholar: https://scholar.google.com/citations?user=7B4GHsMAAAAJ&hl=en

Papers that were not written by me::
2018: 

- [Construction of the Literature Graph in Semantic Scholar](https://aclanthology.org/N18-3011/)

2021: 

- [Improving Biomedical Pretrained Language Models with Knowledge](https://aclanthology.org/2021.bionlp-1.20/)

2022: 

- [Generative Biomedical Entity Linking via Knowledge Base-Guided Pre-training and Synonyms-Aware Fine-tuning](https://aclanthology.org/2022.naacl-main.296/)

- [Fusing Heterogeneous Factors with Triaffine Mechanism for Nested Named Entity Recognition](https://aclanthology.org/2022.findings-acl.250/)

- [BioBART: Pretraining and Evaluation of A Biomedical Generative Language Model](https://aclanthology.org/2022.bionlp-1.9/)

- [Automatic Biomedical Term Clustering by Learning Fine-grained Term Representations](https://aclanthology.org/2022.bionlp-1.8/)

- [Code Synonyms Do Matter: Multiple Synonyms Matching Network for Automatic ICD Coding](https://aclanthology.org/2022.acl-short.91/)

- [CBLUE: A Chinese Biomedical Language Understanding Evaluation Benchmark](https://aclanthology.org/2022.acl-long.544/)

2023:

- [Exploring Partial Knowledge Base Inference in Biomedical Entity Linking](https://aclanthology.org/2023.bionlp-1.3/)

- [Prompt Discriminative Language Models for Domain Adaptation](https://aclanthology.org/2023.clinicalnlp-1.30/)

- [HyPe: Better Pre-trained Language Model Fine-tuning with Hidden Representation Perturbation](https://aclanthology.org/2023.acl-long.182/)

- [The ADAIO System at the BEA-2023 Shared Task: Shared Task Generating AI Teacher Responses in Educational Dialogues](https://aclanthology.org/2023.bea-1.65/)

2024: 

- [Routing to the Expert: Efficient Reward-guided Ensemble of Large Language Models](https://aclanthology.org/2024.naacl-long.109/)

- [Text Diffusion Model with Encoder-Decoder Transformers for Sequence-to-Sequence Generation](https://aclanthology.org/2024.naacl-long.2/)

- [MLDSP-MA: Multidimensional Attention for Multi-Round Long Dialogue Sentiment Prediction](https://aclanthology.org/2024.lrec-main.997/)

**Comment by mjpost** [comment]
Hello: we are attempting to close out a large backlog of author page requests. As part of these efforts, we are collecting additional information ([ORCID](https://orcid.org/) and degree institution) which will help us assign papers to the correct author in the future. Please modify the updated description above with the requested information.

If you are requesting to split an author page (i.e., your page has some papers that are not yours), please also provide a list of your papers, in the form of Anthology IDs or URLS (e.g., 2023.wmt-1.13 or https://aclanthology.org/2023.wmt-1.13/). We are unable to match papers to Google or Semantic Scholar profiles.

**Comment by zhengyuan-nlp** [comment]
Thanks! I’ve added my ORCID, degree institution, and a list of my papers as requested. Let me know if any additional information is needed from my side. Many thanks!

**Comment by mjpost** [comment]
@zhengyuan-nlp Since this is such a large change, do you mind confirming that everything look okay? Here is your preview page: https://preview.aclanthology.org/Author-Page-Zheng-Yuan-cam/people/zheng-yuan-cam/

**Comment by zhengyuan-nlp** [comment]
All looks good to me! Thank you and the team for your help with this -- much appreciated!
